patching/plugin-commands-patching/test/patch.test.ts

```diff
@@ -11,19 +11,25 @@ import { readProjectManifest } from '@pnpm/read-project-manifest'
 import { REGISTRY_MOCK_PORT } from '@pnpm/registry-mock'
 import { DEFAULT_OPTS } from './utils/index'
 
+jest.mock('enquirer', () => ({ prompt: jest.fn() }))
+
+// eslint-disable-next-line
+import * as enquirer from 'enquirer'
+
+// eslint-disable-next-line
+const prompt = enquirer.prompt as any
```

Comment:
there's no need to disable eslint. These import statements can be before `jest.mock`. It'll work.

Reply:
Thanks for your advice, it has been updated